qa/tasks/cephfs/kernel_mount.py

```diff
@@ -67,6 +69,11 @@ def mount(self, mntopts=[], createfs=True, check_status=True, **kwargs):
             if 'permission denied' in stderr.getvalue().lower():
                 pass
 
+        if self.dynamic_debug:
+            kmount_count = self.ctx.get('kmount_count', 0)
+            if kmount_count == 0:
+                self.enable_dynamic_debug()
```

Comment:
Why is doing this once before the first mount necessary?

Reply:
My understanding about `ctx` is that there has a seprate copy for each node (?) and if one node has more than one kernel mounts, we need to make sure the first one will enable it and the last one to disable it.

Reply:
2 kclients could be on 2 nodes where you'd need to enable debugging on both. Presumably, this should work similar to the virtual bridges in terms of setup/teardown.

Reply:
Sorry for late.
Yeah, have renamed the `kmount_count` to `kmount_count.$HOSTNAME` instead.